pythontest.py

```diff
@@ -0,0 +1,20 @@
+#Import Modules
+import json 
+import subprocess
+
+#Open json file and parse its contents inside the data variable using json.load method
```

Comment:
I find it much easier to ready comments if there is a space between the `#` and the first word.